modules/samples/tokenBinding/authorization.php

```diff
@@ -0,0 +1,79 @@
+<?php include "/opt/lampp/htdocs/password/layout/header.php"; ?>
+<?php
+error_reporting(E_ALL ^ E_NOTICE);
+if ($_REQUEST['work'] == "sent"){
+    $curl = curl_init();
+    curl_setopt_array($curl, array(
+        CURLOPT_URL => "https://localhost/oauth2/token",
+        CURLOPT_RETURNTRANSFER => true,
+        CURLOPT_ENCODING => "",
+        CURLOPT_SSL_VERIFYPEER => false,
+        CURLOPT_SSL_VERIFYHOST => 0,
+        CURLOPT_MAXREDIRS => 10,
+        CURLOPT_TIMEOUT => 30,
+        CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
+        CURLOPT_CUSTOMREQUEST => "POST",
+        CURLOPT_POSTFIELDS => "grant_type=" . $_POST['grant_type'] . "&code_verifier=" . $_POST['code_verifier'] . "&code=" . $_POST['code'] . "&redirect_uri=" . $_POST['redirect_uri'] . "&client_id=" . $_POST['client_id'],
+        CURLOPT_HTTPHEADER => array(
+            'authorization: Basic ' . base64_encode($_POST['client_id'] . ":" . $_POST['client_secret']),
+            "cache-control: no-cache",
+            "code_verifier: " . $_POST['code_verifier'],
+            "content-type: application/x-www-form-urlencoded"
+        ),
+    ));
+
+    $response = curl_exec($curl);
+    $err = curl_error($curl);
+
+    curl_close($curl);
+
+    if ($err) {
+        echo "cURL Error #:" . $err;
+    } else {
+        echo $response;
+    }
+}
+else{
```

Comment:
Formatting issue, could you please check the formatting throughput the PR?

Reply:
PR send https://github.com/wso2/product-is/pull/6870